definitions/scenarios/self_upgrade.rb

```diff
@@ -79,6 +79,20 @@ def use_rhsm?
 
       true
     end
+
+    def main_rh_repos
```

Comment:
can't we use the same method from the `downstream` concern?

Reply:
> can't we use the same method from the `downstream` concern?

I felt including whole concern for a method which does not depend anything from downstream concern is too much to include, but yes we can

Reply:
I can see that, but it also means that we have to places to maintain those repo lists, and you know how that will end ;-)